docs/build/get-started.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Get started
+description: Want to jump into Filecoin and start building something? This section gets you up and running quickly by creating a simple Node.js script using Infura and the Filecoin network.
+breadcrumb: 'Get started'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }} The Infura API makes it easier to interact with the Filecoin Network by abstracting away some of the complex workflows and requirements. There's no need to spin up a node or directly call a Lotus daemon!
+
+In this tutorial, you will:
+
+1. Sign up to the Infura API.
+1. Create a Node.js script to:
+   a. Check if a given string is a valid Filecoin address.
+   a. Return the balance of that address if it is valid.
+1. Discover more ways to interact with the Filecoin network using the Infura API.
+
+## Prerequisites
+
+Make sure you have both [Node.js](https://nodejs.org/) and [NPM](https://www.npmjs.com/) installed.
+
+## Infura API
+
+To interact with the Infura API, you need to register on the Infura website. It's a free API, and all you need to do is sign up with your email:
+
+1. Go to [infura.io/register](https://infura.io/register) and sign up.
+1. Select **Filecoin** from the sidebar and click **Create new project**.
```

Comment:
They will first have to join the waitlist. Might be helpful to include this screenshot
![image](https://user-images.githubusercontent.com/40474595/108380914-500e0180-71d5-11eb-9bbd-89762ec4bb10.png)

Reply:
Thanks for this @cheryldouglass. I've [added a call-out on line 11](https://github.com/filecoin-project/filecoin-docs/pull/621/files/9162cd084141367e5462e020b2c298aa6cc0ca09..36aa9e3c5e2cacd472c3b663a676598ec7dc7e66#diff-5d92095ef4c71e27ca46b094bb3fb2996730c55ddbd4b209d7cbe25ee88ed8b8R11) to let users know about the wait-list.